.codespell.ignorewords

```diff
@@ -6,3 +6,4 @@ od
 als
 wit
 aks
+immediatedly
```

Comment:
This and other codespell issues were already fixed in `main` (https://github.com/projectcontour/contour/commit/13c2bd02f6bf4e3cb773e9dd3c01bc3e39d75d45), you should be able to rebase/merge main and have this go away.

Reply:
Thanks. Done.